cuckoo_time_translator_algorithms/CMakeLists.txt

```
@@ -4,12 +4,12 @@ project(cuckoo_time_translator_algorithms)
 find_package(catkin_simple REQUIRED)
 catkin_simple()
 
-find_package(Eigen REQUIRED)
+find_package(Eigen3 REQUIRED)
 find_package(console_bridge REQUIRED)
 
 include_directories(
   ${console_bridge_INCLUDE_DIRS}
-  ${Eigen_INCLUDE_DIRS}
+  ${Eigen3_INCLUDE_DIRS}
```

When I install `libeigen3-dev` (what `rosdep resolve eigen` yields) on Ubuntu Focal, then this must be capital. 

```suggestion
  ${EIGEN3_INCLUDE_DIRS}
```
(see `/usr/lib/cmake/eigen3/Eigen3Config.cmake`). 

I'm quite confused that it works for you with lower case. Are you using some other CMake files (other than from `libeigen3-dev`)?

---

Ah, for sure it should be capitalized. 

Would catkin also play fine with the modern `Eigen3::Eigen` target, instead of separate _LIBRARIES/_INCLUDE_DIRS?